src/main/scala/com/crealytics/spark/excel/ExcelRelation.scala

```diff
@@ -30,9 +31,9 @@ case class ExcelRelation(
 
   lazy val excerpt: List[SheetRow] = workbookReader.withWorkbook(dataLocator.readFrom(_).take(excerptSize).to[List])
 
-  lazy val headerCells = excerpt.head
+  private def headerCells = excerpt.head
```

Comment:
Why did you make that a `def` instead of a `lazy val`?
Does it have to be recomputed?

Reply:
`lazy val` has some runtime overhead with synchronization and `head` and `tail` are just a fields in List, so access is fast.